docs/platform-support.md

```diff
@@ -3,3 +3,4 @@
 FakeItEasy officially supports the following platforms:
 
 * .NET Framework 4.0 onwards
+* .NET Standard 1.6

```

Comment:
Should this line also say "onwards? Or should " onwards" be removed from the previous line?